stunts/stressed/src/app/mainwindow.cpp

```diff
@@ -318,7 +319,7 @@ void MainWindow::about()
   QMessageBox::about(this, tr("About stressed"),
       tr("<div align=\"center\"><h1>%1 %2</h1><br/>"
          "%3<br/>"
-         "Using Qt "QT_VERSION_STR"/%5<br/>"
+         "Using Qt 5.15/%5<br/>"
```

Comment:
Can we still use QT_VERSION_STR as a numbered argument in the tr() format string? `tr("Qt %1").arg(QT_VERSION_STR)`

Reply:
i fixed that in the second pull - its not allowed in C++11 to write `"QT_VERSION_STR"` you need to write `" QT_VERSION_STR "`, (blank before and after `"`) gcc 9.3 warns, VS2019 just don't compiles the line, just add blanks